tests/Adapter/SymfonyCacheTest.php

```diff
@@ -241,4 +269,25 @@ public function testFlushWithIPv6(): void
             $mock->disable();
         }
     }
+
+    /**
+     * @group legacy
+     * @expectedDeprecation Passing no 9th argument to Sonata\CacheBundle\Adapter\SymfonyCache is deprecated since version 3.1 and will be mandatory in 4.0.
```

Comment:
```suggestion
     * @expectedDeprecation Passing no 9th argument to Sonata\CacheBundle\Adapter\SymfonyCache is deprecated since version sonata-project/cache-bundle 3.x and will be mandatory in 4.0.
```